projects/components/src/multi-select/multi-select.component.ts

```diff
@@ -235,11 +236,18 @@ export class MultiSelectComponent<V> implements ControlValueAccessor, AfterConte
 
   public ngAfterContentInit(): void {
     this.allOptions$ = this.allOptionsList !== undefined ? queryListAndChanges$(this.allOptionsList) : EMPTY;
-    this.filteredOptions$ = combineLatest([this.allOptions$, this.caseInsensitiveSearchSubject]).pipe(
+    this.filteredOptions$ = combineLatest([
+      this.allOptions$,
+      this.caseInsensitiveSearchSubject,
+      this.reorderSelectedItemsSubject
```

Comment:
not used.

Reply:
it is used to re-execute the combineLatest to call reordering after click on open. How do you suggest to call this combineLatest without execute a subject?

Reply:
@arjunlalb Can you clarify?

Reply:
@Exac I had discussed this with Pato. We don't need to reorder everytime we open the dropdown. Instead, we should do it on every close. This way, even for large and asynchronous lists, the delay in opening is kept minimal and the reordering happens behind the scenes.